docs/architecture/raft.rst

```diff
@@ -219,6 +219,28 @@ of nodes in the cluster is available. The following examples illustrate how Raft
 
 In summary, Raft makes schema changes safe, but it requires that a quorum of nodes in the cluster is available.
 
+.. _raft-topology-changes:
+
+.. only:: opensource
+
+    Consistent Topology Updates with Raft :label-caution:`Experimental`
+    -------------------------------------------------------------------------------
+
+    ScyllaDB can use Raft for topology updates. With Raft-managed topology enabled, 
+    the cluster can verify that all reachable nodes are using the current topology 
+    and block requests that use the old topology. As a result, topology updates are 
+    both fast and safe, as topology operations can happen in parallel without 
+    risking data inconsistency.
+
+    Support for consistent topology updates with Raft is experimental and must be 
+    explicitly enabled in the ``scylla.yaml`` configuration file by specifying 
+    the ``consistent-topology-changes`` option:
```

Comment:
You can bootstrap a new cluster that way, it's impossible to "upgrade" an existing cluster yet (we're working on the upgrade procedure.)

Reply:
And I would include a warning: `experimental` is not for production clusters, the feature is still undergoing backward incompatible changes. A cluster in `experimental` mode may not be possible to be upgraded later.